java/code/webapp/WEB-INF/pages/channel/all.jsp

```diff
@@ -5,18 +5,15 @@
 
 <html>
 
-<head>
+<body>
 <script src="/javascript/channel_tree.js?cb=${rhn:getConfig('web.version')}" type="text/javascript"></script>
 <script type="text/javascript">
-var filtered = ${requestScope.isFiltered};
-function showFiltered() {
-  if (filtered)
-    ShowAll();
-}
+    var filtered = ${requestScope.isFiltered};
```

Comment:
We might want to instantiate `filtered` at the same time that we call `showFiltered`, instead of doing it earlier?
Btw, at least for me, it's a bit hard to understand that method it supposes to act as a toggle or it might be the naming which confuses me?

Reply:
This is really old code. When the page is refactored to react, we can properly improve and refactor this :)